transport-classes-io_uring/src/main/java/io/netty/channel/uring/AbstractIoUringStreamChannel.java

```diff
@@ -220,26 +222,50 @@ private final class IoUringStreamUnsafe extends AbstractUringUnsafe {
         private ByteBuf readBuffer;
         private IovArray iovArray;
         private IoUringBufferRing lastUsedBufferRing;
+        private int lasSendZCRes;
 
         @Override
         protected int scheduleWriteMultiple(ChannelOutboundBuffer in) {
             assert iovArray == null;
             assert writeId == 0;
-            int numElements = Math.min(in.size(), Limits.IOV_MAX);
-            ByteBuf iovArrayBuffer = alloc().directBuffer(numElements * IovArray.IOV_SIZE);
-            iovArray = new IovArray(iovArrayBuffer);
+            int outstandingWrite = 1;
+            IoRegistration registration = registration();
             try {
-                int offset = iovArray.count();
-                in.forEachFlushedMessage(iovArray);
+                final IoUringIoOps ops;
+                //if the first ByteBuffer in the ChannelOutboundBuffer meets the conditions for sendZC,
+                IoUringIoOps sendZCOps = trySendZC((ByteBuf) in.current());
```

Comment:
Should we only try to use use zero copy if we don't have multiple buffers in the outbound buffer ? I think in this case a single `writev` might be cheaper.

Reply:
I'm not quite sure about this. I think writev is just the batch version of copyable send.

In the case of [bytebuf(writable: 8192), bytebuf(writable: 8)], using sendzc + send for two sqe, I think the sending performance is similar to that of `writev`

If we use writev for packetized sending, it may not fulfill the users' intention of using io_uring_send_zc, especially for those who use the FlushConsolidationHandler.

Reply:
Maybe @axboe has some suggestions